tests/robot-tests/tests/admin/bau/create_data_block_with_chart.robot

```diff
@@ -9,8 +9,9 @@ Suite Setup         user signs in as bau1
 Suite Teardown      user closes the browser
 Test Setup          fail test fast if required
 
-Force Tags          Admin    Local    Dev    AltersData
+Force Tags          Admin    Local    Dev    AltersData    NotAgainstDev
```

Comment:
Since you have "NotAgainstDev" here, I think it makes sense to remove the "Dev" tag.